playhouse/tests/test_fields.py

```diff
@@ -693,3 +693,16 @@ def test_server_default(self):
 
         self.assertEqual(sd2_db.name, 'foo')
         self.assertEqual(sd2_db.timestamp, datetime.datetime(2015, 1, 2, 3, 4))
+
+
+class TestPasswordFields(ModelTestCase):
```

Comment:
Probably need to employ some usage of the ``@skip_if`` decorator. You can see examples in the `test_extra_fields` module.